plan/physical_plan_builder.go

```diff
@@ -69,6 +69,19 @@ func (p *requiredProp) enforceProperty(task task, ctx context.Context) task {
 	return sort.attach2Task(task)
 }
 
+func (p *PhysicalUnionScan) getChildrenPossibleProps(prop *requiredProp) [][]*requiredProp {
+	if !prop.isEmpty() {
```

Comment:
Sort can be pushed down to UnionScan.
refer UnionScanExec.compare

Reply:
ok